subprojects/testfx-core/testfx-core.gradle

```diff
@@ -16,20 +16,46 @@
  */
 ext.pomDescription = 'TestFX Core'
 ext.moduleName = 'org.testfx'
+ext.openjfxVersion = '11-ea+19'
 
 repositories {
     jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
+static def getOSName() {
+    final String osName = System.getProperty("os.name").toLowerCase()
+    if (osName.contains("linux")) {
```

Comment:
Maybe it would be cleaner to use gradle's OperatingSystem interface like this:
https://stackoverflow.com/a/31443955

Reply:
Ah I see! This is how it's done in https://github.com/johanvos/jfx-samples .
Nevermind then. I suspect, it is because it's incubating. I will have another take on this in https://github.com/johanvos/jfx-samples once it's stable.